.releaserc.json

```
@@ -23,10 +29,7 @@
       ]
     }
   ],
-  "publish": [
-    "@semantic-release/npm",
-    "@semantic-release/github"
-  ],
+  "publish": ["@semantic-release/npm", "@semantic-release/github"],
```

A quick search suggests that we should use [distTag](https://docs.npmjs.com/cli/v9/commands/npm-dist-tag) so that we don't set any version to latest. Can you please verify if this makes sense?

So we would have something like:

```suggestion
  "publish": [
  ["@semantic-release/npm", {
    "npmPublish": true,
    "distTag": "v4-latest"
  }], 
  "@semantic-release/github"
],
```